pypfopt/base_optimizer.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, n_assets, tickers=None):
             self.tickers = tickers
         # Outputs
         self.weights = None
+        self.verbose = False
```

Comment:
I added the instance variable to the constructor but made it _really_ hidden by not including it in the `initialize` signature. If that is _too_ hidden, lmk and I will update to
``` Python
#  def __init__(self, n_assets, tickers=None):
def __init__(self, n_assets, tickers=None, verbose=False):
```